Packs/epo/ReleaseNotes/2_0_2.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### McAfee ePO v2
+###### Command *epo-advanced-command*
```

Comment:
Follow the release-notes convention 
https://xsoar.pan.dev/docs/documentation/release-notes#bug-fixes-examples

Reply:
@Shellyber  I really appreciate your input here I have resolved both of them in the last commit?